src/web/mjs/connectors/Comikey.mjs

```diff
@@ -0,0 +1,66 @@
+import Connector from '../engine/Connector.mjs';
+
+export default class ComiKey extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'comikey';
+        super.label = 'Comikey';
+        this.tags = ['manga', 'webtoon', 'english'];
+        this.url = 'https://comikey.com';
+        this.chapterUrl = 'https://relay-us.epub.rocks/consumer/COMIKEY/series/';
```

Comment:
Is the `US` server acceptable worldwide, or maybe use `EU` => `relay-eu` instead?

Reply:
Seems to be region specific routing. relay-eu is also a valid path. US server seems acceptable worldwide as i got the response from the API in EU network zone.